src/vm/gdbjit.cpp

```diff
@@ -1739,7 +1739,7 @@ void NotifyGdb::MethodCompiled(MethodDesc* MethodDescPtr)
     {
         cCharsNeeded = GetEnvironmentVariableW(W("CORECLR_GDBJIT"), NULL, 0);
 
-        if((cCharsNeeded == 0) || (cCharsNeeded >= MAX_LONGPATH))
+        if(cCharsNeeded == 0)
             return;
         wszModuleNames = new WCHAR[cCharsNeeded+1];
         cCharsNeeded = GetEnvironmentVariableW(W("CORECLR_GDBJIT"), wszModuleNames, cCharsNeeded);
```

Comment:
@chunseoklee I meant the wszModuleNames. But I have not noticed before that this new is not nothrow, so the nullptr check is not missing. I can see that while most of the allocations in this file use `new (nothrow)`, about 14 of them don't. We should unify that.
So this change is ok and I'll create a new issue to fix the allocations.